.github/workflows/build_and_publish.yml

```diff
@@ -0,0 +1,55 @@
+name: Build and upload to PyPI
+
+on:
+  pull_request:
+  release:
+    types:
+      - published
+
+jobs:
+  build_sdist:
+    name: Build source distribution
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Build sdist
+        run: pipx run build --sdist
```

Comment:
I found this step in `cibuildwheels` docs, and the isolated environment provided by pipx made sense to me at first glance. I can check out `hatch` this Monday.

Reply:
There was a bit of confusion about this line. `pipx` is just providing an isolated environment for `build`, which is a package separate from `pipx`, and made for creating distributions.

Reply:
@RokasUrbonasQC Did @ivergara tell you to resolve this or do you feel very certain to have addressed his concern?

Reply:
Sorry, I think my last comment didn't give enough context. We had a short discussion on Slack about this with Ignacio. My message detailing that `build` is a separate package rather than a part of `pipx` was followed by Ignacio's reply saying: "Then all good". I decided that this response was enough to call the issue resolved.